core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -302,18 +371,21 @@ public Put withBlobValue(String columnName, ByteBuffer value) {
    */
   @Deprecated
   public Put withValues(Collection<Value<?>> values) {
-    values.forEach(v -> this.values.put(v.getName(), Optional.of(v)));
+    values.forEach(v -> this.columns.put(v.getName(), ScalarDbUtils.toColumn(v)));
     return this;
   }
 
   /**
-   * Adds NULL value to the list of put values.
+   * Adds a column to the list of put values.
    *
-   * @param columnName a column name of the value
+   * <p>This method is primarily for internal use. Breaking changes can and will be introduced to
+   * this method. Users should not depend on it.
+   *
+   * @param column a column to put
    * @return this object
    */
-  public Put withNullValue(String columnName) {
```

Comment:
As mentioned, how to specify NULL values is changed in this PR, so I removed this `withNullValue()`.